src/components/program/programPlanAdminModals/formItem/TitleItem.tsx

```diff
@@ -0,0 +1,33 @@
+import { Form, Input } from 'antd'
+import { useIntl } from 'react-intl'
+import { errorMessages } from '../../../../helpers/translation'
+import programMessages from '../../translation'
+
+interface TitleProps {
+  label?: string
+  name: string
+}
+
+const TitleItem: React.FC<TitleProps> = ({ label, name }) => {
+  const { formatMessage } = useIntl()
+  const _label = label ? label : formatMessage(programMessages.ProgramPlanAdminModal.planTitle)
```

Comment:
Keeping flexibility is very beneficial

Reply:
FYR: https://www.patterns.dev/react/compound-pattern/

Reply:
it is better to use `formItems/` instead of `formItem/`